### DISCUSSION TITLE
Changes included in latest experimental build

### DISCUSSION THREAD

**Discussion by bucmic04** [flybywiresim/aircraft#7892] [General]
Ho do I find what what changes are included in the latest experimental build?

**Comment by tomprovence2** [comment]
https://github.com/flybywiresim/a32nx/actions/workflows/experimental.yml

**Reply by tomprovence2** [reply]
https://github.com/flybywiresim/a32nx/commits/experimental

**Reply by bucmic04** [reply]
Thanks. So the exp build runs on every PR in the exp branch.
